api/instance/v1/instance_utils.go

```diff
@@ -303,3 +295,24 @@ func (r *ListServersTypesResponse) UnsafeAppend(res interface{}) (int, scw.SdkEr
 	r.TotalCount += uint32(len(results.Servers))
 	return len(results.Servers), nil
 }
+
+func (v *NullableStringValue) UnmarshalJSON(b []byte) error {
+	if string(b) == "null" {
+		v.Null = true
+		return nil
+	}
+
+	var tmp string
+	if err := json.Unmarshal(b, &tmp); err != nil {
+		return err
+	}
+	v.Value = tmp
```

Comment:
You should add `v.Null = false` as someone could reuse an already used object while Unmarshaling

Reply:
Good catch!